Calculator.c

```diff
@@ -0,0 +1,111 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<string.h>
+#include<ctype.h>
+#define MAX 80
+
+int cal(int op1,int op2,char op){
+    if(op=='+'){
+        return op1+op2;
+    }
+    else if(op=='-'){
+        return op1-op2;
+    }
+    else if(op=='*'){
+        return op1*op2;
+    }
+    else if(op=='/'){
+        if(op2==0){
+            printf("Division by Zero error");
+            exit(1);
+        }
+        return op1/op2;
+    }
+    else{    
+        printf("Error:Invalid Operator");
+        exit(1);
+    }
+}
+
+
+int evalvate(char* exp){
+    int num_stack[MAX];
+    char op_stack[MAX];
+    int num_top = -1;int op_top = -1;
+    int length = strlen(exp);
```

Comment:
Instead of using built-in functions like strlen() and isspace(), Can you create your own definition of these functions and remove unnecessary headerfile?